packages/dashboard-hardhat-plugin/test/fixture-projects/hardhat-project-incompatible/contracts/SimpleStorage.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.21 <0.9.0;
+
+contract SimpleStorage {
+    event StorageSet(string _message);
+
+    uint256 public storedData;
+
+    function set(uint256 x) public {
+        storedData = x;
+
+        emit StorageSet("Data stored successfully!");
+    }
+}
```

Comment:
Missing new line at end of file (just turn this on in your editor :D)

Reply:
I found this real quick on how to do this with VS Code (although I haven't tried it) https://stackoverflow.com/a/44704969

Reply:
Awesome, that works a treat 👍

Reply:
New line added in 4b024e8f12d7ed27835e9c76b47f0f41a6c9f388 👍